applications/nrf_desktop/configuration/nrf52840dk_nrf52840/pm_static_fast_pair.yml

```diff
@@ -1,43 +1,51 @@
-b0_container:
+app:
+  address: 0x8200
+  region: flash_primary
+  size: 0x79e00
+mcuboot:
   address: 0x0
+  region: flash_primary
+  size: 0x8000
```

Comment:
remember to reserve some room for the MCUboot partition so that we don't run into memory issues as the kernel and bootloader grow in size over time.

Reply:
Done already. See the memory reports in following comments.
The nRF52840 SoC seems to require bigger MCUboot partition because it uses CryptoCell for crypto operations by default, which noticeably increases MCUboot's FLASH usage comparing to nRF52832 SoC.

Reply:
nRF52840DK:
Memory region         Used Size  Region Size  %age Used
           FLASH:       23604 B        32 KB     72.03%
             RAM:       19288 B       256 KB      7.36%
        IDT_LIST:          0 GB         2 KB      0.00%

Reply:
nRF52840 gaming mouse:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:       23436 B        32 KB     71.52%
             RAM:       19288 B       256 KB      7.36%
        IDT_LIST:          0 GB         2 KB      0.00%
```

Reply:
nRF52 Keyboard
```
Memory region         Used Size  Region Size  %age Used
           FLASH:       19976 B        24 KB     81.28%
             RAM:       19168 B        64 KB     29.25%
        IDT_LIST:          0 GB         2 KB      0.00%
```